pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -0,0 +1,234 @@
+package policy
+
+import (
+	"errors"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
+)
+
+const (
+	AddSCCToGroupRecommendedName      = "add-scc-to-group"
```

Comment:
> These seem backwards to me since you're adding group/user to the scc

As opposed to granting the powers of the SCC to the user?  We face this with roles as well.  Do you add a user to a role or do you add a role to user and we chose this direction.  Do you think SCCs different enough to reverse the usage?